description.md

```diff
@@ -36,6 +36,32 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Exams
 
+Exams are assessments designed to evaluate learners' understanding and application of the course material.
+Exams aim to measure comprehension, critical thinking, and problem-solving skills.
+
+Examinations can take various forms, including:
+
+- **Written Exams** consist of short answers, essays, or problems to solve.
+They are versatile and work for various subjects. Written exams build on what you've learned earlier.
+Written exams build upon previous learning, but they aren't adaptive and don't personalize the experience for each student.
+Additionally, they might not be the best choice for assessing complex situations, often leaning towards straightforward questions.
+
+- **Multiple-Choice Exams** involve choosing the correct answer from a set of options, essentially being a compilation of [quizzes](#quizzes).
+You can consider creating two sets of quizzes: one accessible to all for learning purposes, and another separate set reserved for assessment.
+However, this approach is only a short-term solution, as questions may eventually become accessible through leaks.
+Our recommendation leans toward making all quizzes publicly available and concentrating on producing a substantial number of them.
+This strategy encourages students to engage with the quizzes in the hope of identifying possible exam-related queries.
+
+* **Oral Exams**: Students respond verbally to questions from the instructor to demonstrate understanding.
```

Comment:
Some oral exams involve solving problems on a white/blackboard. They may be useful for mathematical and theoretical subjects (Maths, Physics, maybe Chemistry). You should mention those.